src/services/commandService.ts

```diff
@@ -1,3 +1,23 @@
-export function parse(rawCommand: string): {} {
-    return { raw: rawCommand }
+export function parseGive(rawCommand: string): {} {
+    const kudosReceiverId = getUserIdFromRaw(rawCommand)
+    const kudosTransactionValue = getKudosValueFromRaw(rawCommand)
+    return {
+        kudosReceiverId: kudosReceiverId,
+        kudosTransactionValue: kudosTransactionValue
+    }
```

Comment:
You can shorten that by:
`return {
        kudosReceiverId,
        kudosTransactionValue
    }`